Gemfile.lock

```diff
@@ -247,3 +247,6 @@ DEPENDENCIES
   simplecov
   timecop
   yard
+
```

Comment:
Apologies. What I'm saying is that Bundler 1.10.0 through 1.10.3 do a silly thing and add this 'bundled by' line automatically, when you actually don't need to change this file for this PR. I will elide it. If you update your local environment to Bundler 1.10.5, it will stop adding this on its own.

No worries, I fix this bit up when this lands.